src/app/community-view/community-view.component.spec.ts

```diff
@@ -188,4 +190,33 @@ describe('CommunityViewComponent', () => {
     fixture.detectChanges();
     expect(component.isCommunityMember).toBeFalsy();
   }));
+
+  it('should not make user join a closed community', fakeAsync(() => {
```

Comment:
I suggest to add `... and display closed community info dialog` here because the test checks if the dialog has been displayed.

Reply:
done